bin/i18n/translation_status/translation_service.rb

```diff
@@ -12,8 +12,8 @@ def initialize
   end
 
   # Returns true if a translation exists for a given key, otherwise returns false.
-  def translated?(locale, key)
-    I18n.exists?(key, locale: locale)
+  def translated?(locale, key, scope)
+    scope == "" ? I18n.exists?(key, locale: locale) : !I18n.t(key, scope: JSON.parse(scope), smart: true).include?('translation missing:')
```

Comment:
Consider using `I18n.exists?` with fallback disabled https://github.com/ruby-i18n/i18n/pull/482

Reply:
Unfortunately we're on i18n v1.6.0, and that PR wasn't merged until v1.8.3. Can we get a ticket in the backlog to upgrade our i18n version and note that we should disable fallback here when that work is completed? Or do we want to block this work on that upgrade?

Reply:
Per slack discussion we've decided to upgrade and disable fallback after this merges